classes/controller/FrontController.php

```diff
@@ -1077,6 +1077,8 @@ public function registerStylesheet($id, $relativePath, $params = [])
             'priority' => AbstractAssetManager::DEFAULT_PRIORITY,
             'inline' => false,
             'server' => 'local',
+            'version' => null,
+            'needRtl' => true
```

Comment:
```suggestion
            'needRtl' => true,
```

Reply:
@PululuK thank you

Reply:
I use my custom php-cs-fixer script https://gist.github.com/PululuK/07ac93607dd6a30c90f9034f32117dcd